docs/tools/vdb_table/data/qdrant.json

```diff
@@ -73,18 +73,18 @@
     "comment": "via FastEmbed (Supported models: https://qdrant.github.io/fastembed/examples/Supported_Models/)"
   },
   "embeddings_image": {
-    "support": "",
+    "support": "full",
     "source_url": "",
-    "comment": ""
+    "comment": "Bring-your-own-embeddings"
   },
   "embeddings_structured": {
     "support": "",
     "source_url": "",
     "comment": ""
   },
   "rag": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://colab.research.google.com/github/qdrant/examples/blob/master/rag-openai-qdrant/rag-openai-qdrant.ipynb",
```

Comment:
This column is for "Calls LLM internally for RAG". Every VDB would be able to call OpenAI as in the notebook you have shared.